### ISSUE TITLE
Reproducing WRN-28-10 (SAM) for SVHN dataset

### ISSUE THREAD

**Issue by akshaydp1995** [sayakpaul/Sharpness-Aware-Minimization-TensorFlow#4]
I am trying to reproduce the results for WRN-28-10 (SAM) trained on 10-class classification SVHN dataset (Percentage Error 0.99) - https://paperswithcode.com/sota/image-classification-on-svhn  

I'm able to train WRN-28-10 using https://github.com/hysts/pytorch_wrn (Modified the script to incorporate SAM into it)

I'm achieving a test accuracy of 93%. How can I replicate the SOTA Percentage Error 0.99 for WRN-28-10 (SAM). Which hyperparameters do I use?

Any help is appreciated!!

**Comment by sayakpaul** [comment]
Contact the paper authors for that or check out the official SAM code.
